README.md

```diff
@@ -16,6 +16,13 @@ on improved performance.
 curl -sfL https://raw.githubusercontent.com/ducaale/xh/master/install.sh | sh
 ```
 
+### via Powershell (Windows)
+
+```
+  PowerShell -ExecutionPolicy bypass -Command "iwr -useb https://raw.githubusercontent.com/ducaale/xh/master/install.ps1 | iex"
```

Comment:
It is not. I tested this on a virtual machine with clean windows for prompts. 
It can be removed, so: `PowerShell "iwr -useb https://raw.githubusercontent.com/ducaale/xh/master/install.ps1 | iex"`

Reply:
Can we make the command even shorter?

```
# Deno
iwr https://deno.land/install.ps1 -useb | iex

# Scoop
iwr -useb get.scoop.sh | iex

# Chocolatey
iwr https://chocolatey.org/install.ps1 -UseBasicParsing | iex
```

Reply:
So it is still required to have scoop?

Reply:
No, I was just listing some existing install instructions that don't start with `PowerShell -ExecutionPolicy bypass -Command`.

Reply:
Powershell part is indeed only required when running command from Run and the command to run in Powershell is between quotes. In Powershell it is not needed.

Reply:
Let's assume the user will run the command from Powershell.